diff -ur nautilus-dropbox-2020.03.04/dropbox.in nautilus-dropbox-2020.03.04-python3.10/dropbox.in
--- nautilus-dropbox-2020.03.04/dropbox.in	2020-03-04 23:49:25.000000000 +0100
+++ nautilus-dropbox-2020.03.04-python3.10/dropbox.in	2022-06-22 13:51:30.818855370 +0200
@@ -611,7 +611,7 @@
         first = True
         while True:
             self.stop_event.wait(0.25)
-            if self.stop_event.isSet(): break
+            if self.stop_event.is_set(): break
             if i == len(ticks):
                 first = False
                 i = 0
