diff -uNrp a/src/dropbox-command-client.h b/src/dropbox-command-client.h
--- a/src/dropbox-command-client.h	2019-01-10 01:12:50.000000000 +0000
+++ b/src/dropbox-command-client.h	2019-02-12 08:52:27.443189961 +0000
@@ -24,8 +24,7 @@
 #ifndef DROPBOX_COMMAND_CLIENT_H
 #define DROPBOX_COMMAND_CLIENT_H
 
-#include <libnautilus-extension/nautilus-info-provider.h>
-#include <libnautilus-extension/nautilus-file-info.h>
+#include <nautilus-extension.h>
 
 G_BEGIN_DECLS
 
