diff -uNrp a/src/dropbox-command-client.h b/src/dropbox-command-client.h
--- a/src/dropbox-command-client.h	2019-01-10 01:12:50.000000000 +0000
+++ b/src/dropbox-command-client.h	2019-02-12 08:52:27.443189961 +0000
@@ -24,8 +24,7 @@
 #ifndef DROPBOX_COMMAND_CLIENT_H
 #define DROPBOX_COMMAND_CLIENT_H
 
-#include <libnautilus-extension/nautilus-info-provider.h>
-#include <libnautilus-extension/nautilus-file-info.h>
+#include <nautilus-extension.h>
 
 G_BEGIN_DECLS
 
diff -uNrp a/src/nautilus-dropbox.c b/src/nautilus-dropbox.c
--- a/src/nautilus-dropbox.c	2019-01-10 01:12:50.000000000 +0000
+++ b/src/nautilus-dropbox.c	2019-02-12 08:51:10.385813888 +0000
@@ -37,9 +37,7 @@
 #include <glib-object.h>
 #include <gtk/gtk.h>
 
-#include <libnautilus-extension/nautilus-extension-types.h>
-#include <libnautilus-extension/nautilus-menu-provider.h>
-#include <libnautilus-extension/nautilus-info-provider.h>
+#include <nautilus-extension.h>
 
 #include "g-util.h"
 #include "dropbox-command-client.h"
diff -uNrp a/src/nautilus-dropbox.h b/src/nautilus-dropbox.h
--- a/src/nautilus-dropbox.h	2019-01-10 01:12:50.000000000 +0000
+++ b/src/nautilus-dropbox.h	2019-02-12 08:51:57.680430947 +0000
@@ -27,7 +27,7 @@
 #include <glib.h>
 #include <glib-object.h>
 
-#include <libnautilus-extension/nautilus-info-provider.h>
+#include <nautilus-extension.h>
 
 #include "dropbox-command-client.h"
 #include "nautilus-dropbox-hooks.h"
