--- a/configure.ac
+++ b/configure.ac
@@ -29,16 +29,16 @@
 PKG_CHECK_MODULES(NAUTILUS, libnautilus-extension >= $NAUTILUS_REQUIRED)
 PKG_CHECK_MODULES(GLIB, glib-2.0 >= $GLIB_REQUIRED)
 
-AC_PATH_PROG([PYTHON], [python])
+AC_PATH_PROG([PYTHON], [python2])
 
-AC_PATH_PROG([RST2MAN], [rst2man], [python rst2man.py])
+AC_PATH_PROG([RST2MAN], [rst2man], [python2 rst2man.py])
 AC_SUBST(RST2MAN)
 
 # define module checking macro
 AC_DEFUN([PYTHON_CHECK_MODULE], [
 AC_MSG_CHECKING([for $1])
 
-cat <<EOF | python
+cat <<EOF | python2
 try:
  import $2
 except:

--- a/dropbox.in
+++ b/dropbox.in
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 #
 # Copyright (c) Dropbox, Inc.
 #

--- a/rst2man.py
+++ b/rst2man.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2
 
 # Author: 
 # Contact: grubert@users.sf.net

--- a/Makefile.am
+++ b/Makefile.am
@@ -6,11 +6,11 @@
 man_MANS = dropbox.1
 
 dropbox: dropbox.in serializeimages.py
-	python serializeimages.py $(PACKAGE_VERSION) $(datadir)/applications < dropbox.in > dropbox
+	python2 serializeimages.py $(PACKAGE_VERSION) $(datadir)/applications < dropbox.in > dropbox
 	chmod +x dropbox
 
 dropbox.1: dropbox.txt.in dropbox docgen.py
-	python docgen.py $(PACKAGE_VERSION) < dropbox.txt.in > dropbox.txt
+	python2 docgen.py $(PACKAGE_VERSION) < dropbox.txt.in > dropbox.txt
 	$(RST2MAN) dropbox.txt > dropbox.1
 
 SUBDIRS = data src

